website/docs/components/form/masked-input/partials/code/how-to-use.md

```diff
@@ -0,0 +1,253 @@
+  <F.Label>Terraform Cloud team token</F.Label>
+</Hds::Form::MaskedInput::Field>
+```
+
+#### Events handling
+
+Because this component supports use of `...attributes`, you can use all the usual Ember techniques for event handling (e.g., `input`, `blur`, `change`), validation, etc.
+
+```handlebars
+<Hds::Form::MaskedInput::Field {{on "blur" this.yourOnBlurFunction}} as |F|>
+  <F.Label>Terraform Cloud team token</F.Label>
+</Hds::Form::MaskedInput::Field>
+```
+
+#### Custom width
+
+By default, the input control width is set to fill the parent container.
+
+Pass a custom width for the control using the `@width` argument.
+
+```handlebars
+<Hds::Form::MaskedInput::Field @width="250px" as |F|>
+  <F.Label>Terraform Cloud team token</F.Label>
+</Hds::Form::MaskedInput::Field>
+```
+
+{{! ================= }} {{! ===== BASE ===== }} {{! ================= }}
+
+### Form::MaskedInput::Base
+
+The Base component is intended for rare cases where the Field component can’t be used and a custom implementation is needed. Most of the details for the Field component also apply to the Base component, but see the [Component API](#component-api) for more details.
+
+!!! Warning
+
+`Form::MaskedInput::Base` does not come with built-in accessibility functionality. It is the responsibility of the product team to ensure the implementation is conformant.
+!!!
+
+The default invocation creates a `<input type="text">` or a `<textarea>` control with an automatically generated `ID` attribute.
+
+```handlebars
+<Hds::Form::MaskedInput::Base
+  @value="036215df4996ca649928d8864b4df9e42cba0d6d"
+  aria-label="Terraform Cloud team token"
+/>
+```
+
+When `@isMultiline` argument is set to `true` it creates a `<textarea>` control with an automatically generated `ID` attribute. You can also adjust the height of `<textarea>` either by using the `rows` attribute or setting a custom `@height` value.
+
+```handlebars
+<Hds::Form::MaskedInput::Base
+  @value="-----BEGIN RSA PRIVATE KEY-----
+MIIBOgIBAAJBAKj34GkxFhD90vcNLYLInFEX6Ppy1tPf9Cnzj4p4WGeKLs1Pt8Qu
+KUpRKfFLfRYC9AIKjbJTWit+CqvjWYzvQwECAwEAAQJAIJLixBy2qpFoS4DSmoEm
+o3qGy0t6z09AIJtH+5OeRV1be+N4cDYJKffGzDa88vQENZiRm0GRq6a+HPGQMd2k
+TQIhAKMSvzIBnni7ot/OSie2TmJLY4SwTQAevXysE2RbFDYdAiEBCUEaRQnMnbp7
+9mxDXDf6AU0cN/RPBjb9qSHDcWZHGzUCIG2Es59z8ugGrDY+pxLQnwfotadxd+Uy
+v/Ow5T0q5gIJAiEAyS4RaI9YG8EWx/2w0T67ZUVAw8eOMB6BIUg0Xcu+3okCIBOs
+/5OiPgoTdSy7bcF9IGpSE8ZgGKzgYQVZeN97YE00
+-----END RSA PRIVATE KEY-----"
+  @isMultiline={{true}}
+  rows="5"
+  aria-label="Private key"
+/>
+```
+
+!!! Info
+
+**Important to know**
+
+When the multiline input is masked, the browser converts newline characters to masked characters: this means that the multiline text will appear as a single long string of characters, even if it's inside a `<textarea>` element.
+
+Instead, when the text is not masked it will respect the newline characters: this means it may occupy more lines that when it’s masked (try the example above).
```

Comment:
Same feedback as above

```suggestion
When the text is not masked the newline characters will be respected. This means it may occupy more lines than when it’s masked (see the example above).
```